content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Configuring Azure AD as the SAML IDP with Greenlake Cloud Platform and Aruba Central
+date: 2022-05-09T12:04:30.750Z
+author: Michael Rose Jr.
+authorimage: /img/mrose-photoprofile.png
+tags:
+  - aruba
+  - aruba-central
+---
+
+Single sign-on (SSO) enables users to securely authenticate with multiple applications and websites by logging in only once—with just one set of credentials (username and password). With SSO, the application or website that the user is trying to access relies on a trusted third party (Identity provider) to verify that users are who they say they are.
+
+I'll explain the process for configuring Azure AD to authenticate users into Greenlake Cloud Platform (GLCP) and Aruba Central using SAML IDP.
+
+If you're looking for the Okta version of this guide can be found on [WIFI-GUYS](https://www.wifi-guys.com/?p=512)
+<!-- prettier-ignore-start -->
+## Contents
+
+- [Configuring Azure AD as the SAML IDP with Greenlake Cloud Platform and Aruba Central](#configuring-azure-ad-as-the-saml-idp-with-greenlake-cloud-platform-and-aruba-central)
+	- [Contents](#contents)
+	- [Before you Begin](#before-you-begin)
+	- [Terms used in this blog post](#terms-used-in-this-document)
+	- [Steps to Configure SSO/SAML Application in Azure AD](#steps-to-configure-ssosaml-application-in-azure-ad)
+	- [Step 1: Create an Azure AD Enterprise Application](#step-1-create-an-azure-ad-enterprise-application)
+	- [Step 2: Configure GCLP for SAML Federation](#step-2-configure-gclp-for-saml-federation)
+	- [Login to GLCP and Aruba Central using Azure AD](#login-to-glcp-and-aruba-central-using-azure-ad)
+	- [Using Azure AD MFA](#using-azure-ad-mfa)
+	- [Troubleshooting](#troubleshooting)
+	- [Appendix: Generating the hpe_ccs_attribute](#appendix-generating-the-hpe_ccs_attribute)
+<!-- prettier-ignore-end -->
+
+## Before you Begin
+This blog references the following documentation:
```

Comment:
add "post" after "blog". (A blog is a website, essentially an online magazine. A post is the actual article. So it would read: This blog post references the following documentation.)